File: rust/imzero2/imzero2_egui/src/style/tokens/rounding.rs
//! Rounding tokens (ADR-0032 §SD3).
//!
//! Density-independent — rounding is aesthetic identity, not perceptual
//! magnitude. Swiss-minimalist defaults lean toward `NONE` and `SM`; `LG`
//! is reserved for floating windows that need visual separation from
//! screen edges.

/// Sharp corners. The Swiss default for most surfaces.
pub const NONE: f32 = 0.0;
/// Subtle softening: buttons, badges, inline chips.
pub const SM: f32 = 2.0;
/// Cards, dialogs, panels.
pub const MD: f32 = 4.0;
/// Floating windows, modals, popovers.
pub const LG: f32 = 6.0;

File: public/keelson/designsystem/styletokens/rounding.go
//go:build llm_generated_opus47

package styletokens

// Rounding scale (ADR-0032 §SD3). Density-independent.
//
// MIRROR INVARIANT: must equal Rust consts in
// src/rust/imzero2_egui/src/style/tokens/rounding.rs.
const (
	// RoundingNone — sharp corners, the Swiss default for most surfaces.
	RoundingNone float32 = 0.0
	// RoundingSm — subtle softening: buttons, badges, inline chips.
	RoundingSm float32 = 2.0
	// RoundingMd — cards, dialogs, panels.
	RoundingMd float32 = 4.0
	// RoundingLg — floating windows, modals, popovers.
	RoundingLg float32 = 6.0
)
